VENDORING.md

```diff
@@ -0,0 +1,238 @@
+# Perform the vendoring of the Kata Containers project
+
+* [Vendoring tool](#vendoring-tool)
+    * [How to install dep](#how-to-install-dep)
+    * [Overview of the main commands](#overview-of-the-main-commands)
+* [Vendoring use cases](#vendoring-use-cases)
+    * [Initialize vendoring of the repository](#initialize-vendoring-of-the-repository)
+    * [Vendor a newly imported package](#vendor-a-newly-imported-package)
+    * [Update a vendored package](#update-a-vendored-package)
+    * [Remove a vendored package](#remove-a-vendored-package)
+    * [Test a pending pull request](#test-a-pending-pull-request)
+
+## Vendoring tool
+
+[`dep`](https://github.com/golang/dep) is the vendoring tool chosen for this
+project. The choice has been made to use this tool because it is fairly simple
+and very powerful for some corner cases. Furthermore, this tool is expected to
+become the official vendoring tool for Golang.
+
+### How to install dep
+
+Installing this tool is simple, use the following command:
+```bash
+go get -u github.com/golang/dep/cmd/dep
+```
+
+Make sure your `PATH` has the path `$GOPATH/bin`, including all Go binaries,
```

Comment:
How about simplifying this to something like,

> Make sure you `PATH` contains `$GOPATH/bin`:

Reply:
done